src/components/drawer/ApplicationDrawer.vue

```diff
@@ -43,6 +43,11 @@ const menuList = ref([
     label: t('ApplicationDrawer.text.users'),
     url: '/users',
   },
+  {
```

Comment:
add a variable `name` to each item of the list to use it in `data-cy` instead of `label`